src/clients/github.js

```diff
@@ -143,20 +124,20 @@ export function createGitHubRepository(projectName, projectDescription, token) {
  * @param projectName
  */
 export function initAddCommitAndPush(username, projectName, isTwoFactorAuth) {
-  winston.log('verbose', 'initAddCommitAndPush', { username, projectName });
-  winston.log('info', 'Pushing all files to the new git repository...');
+  winston.log('verbose', 'initialize github repo, create repo and push to repo', { username, projectName });
+  console.log(chalk.yellow('Pushing all files to the new git repository...'));
 
   return new Promise((resolve) => {
     if (!isTwoFactorAuth) {
       git(`${process.cwd()}/${projectName}`)
         .init()
         .add('.gitignore')
         .add('./*')
-        .commit('Initial commit')
+        .commit('Initial Commit w/ :heart: by @hammer-io.')
```

Comment:
The changes here and below (line 150) reveal some duplicate code. Is it possible to move the `git.init()...commit()` outside the if-else block?

Reply:
@KrashLeviathan, I tried doing this once before. It did not work for me. I'll leave it as is for now, but we can open an issue for further research or just leave it.

Reply:
:+1: